rnaseq.nf

```diff
@@ -13,7 +13,7 @@ params.summary_params = [:]
  */
 // Check if genome exists in the config file
 if (params.genomes && params.genome && !params.genomes.containsKey(params.genome)) {
-    exit 1, "The provided genome '${params.genome}' is not available in the iGenomes file. Currently the available genomes are ${params.genomes.keySet().join(", ")}"
+    exit 1, "Genome '${params.genome}' was not found in any config files made available to the pipeline. Currently, the available genomes are ${params.genomes.keySet().join(", ")}"
```

Comment:
This was reported in https://github.com/nf-core/atacseq/issues/126 but fixing here because it will eventually be propagated to the pipeline template and then out to each of these pipelines when the DSL2 pipeline template is eventually released.